HHS/OPRE-OPS
# feat: adds CAN detail page

## What changed

- adds CAN detail page

## Issue

#2542 

## How to test

1. goto CANs list
2. select FY 2023 from dropdown
3. select a CAN from table
4. see CAN detail page

## Definition of Done Checklist
